content/en/tracing/setup_overview/open_standards/otel_collector_datadog_exporter.md

```diff
@@ -98,28 +89,34 @@ service:
 
 Run an Opentelemetry Collector container to receive traces either from the [installed host](#receive-traces-from-host), or from [other containers](#receive-traces-from-other-containers).
 
+<div class="alert alert-info">
+The latest tag of the OpenTelemetry Collector Contrib distro <a href="https://github.com/open-telemetry/opentelemetry-collector-releases/issues/73">is not updated on every release</a>.
+Pin the Collector to the latest version to pick up the latest changes.
+</div>
+
 #### Receive traces from host
 
 1. Create a `otel_collector_config.yaml` file. [Here is an example template](#ingesting-opentelemetry-traces-with-the-collector) to get started. It enables the collector's OTLP receiver and the Datadog exporter.
 
 2. Choose a published Docker image such as [`otel/opentelemetry-collector-contrib:latest`][12].
 
-3.  Determine which ports to open on your container. OpenTelemetry traces are sent to the OpenTelemetry Collector over TCP or UDP on several ports, which must be exposed on the container. By default, traces are sent over OTLP/gRPC on port `55680`, but common protocols and their ports include:
+3.  Determine which ports to open on your container. OpenTelemetry traces are sent to the OpenTelemetry Collector over TCP or UDP on several ports, which must be exposed on the container. By default, traces are sent over OTLP/gRPC on port `4317`, but common protocols and their ports include:
 
       - Zipkin/HTTP on port `9411`
       - Jaeger/gRPC on port `14250`
       - Jaeger/HTTP on port `14268`
       - Jaeger/Compact on port (UDP) `6831`
-      - OTLP/gRPC on port `55680`
+      - OTLP/gRPC on port `4317`
```

Comment:
The default port was changed to 4317 (a long time ago!) https://github.com/open-telemetry/opentelemetry-specification/issues/1148#issuecomment-742080390